fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAboutRobot.kt

```diff
@@ -58,6 +59,12 @@ class SettingsSubMenuAboutRobot {
             return SettingsRobot.Transition()
         }
     }
+
+    fun enableDebugMenu() {
+        for (i in 1..6) {
+            onView(ViewMatchers.withId(R.id.wordmark)).click()
+        }
+    }
```

Comment:
I'd prefer if we use the app context to enable the debug menu directly in your test instead of this function:

`import org.mozilla.fenix.helpers.TestHelper` in your test
and call 
`context = TestHelper.appContext`
`context.settings().showSecretDebugMenuThisSession = true`
and set this in the `@Before` `setUp` method.

Reply:
I have implemented this